react-native-shimmer.podspec

```diff
@@ -1,5 +1,7 @@
 require 'json'
-version = JSON.parse(File.read('package.json'))["version"]
+package = JSON.parse(File.read('package.json'))
+version = package["version"]
+react_native_version = package["peerDependencies"]['react-native']
```

Comment:
This one is unused afaict

Reply:
Thanks for catching that -- I had tried to match this with the RN in `package.json`, but the version specifiers have slightly different syntax, so I gave up.